docs/guide/db-active-record.md

```diff
@@ -1350,41 +1349,44 @@ class Comment extends ActiveRecord
         return new CommentQuery(get_called_class());
     }
 }
+````
+Now whenever you are performing a query (e.g. `find()`, `findOne()`) or defining a relation (e.g. `hasOne()`)
+with `Comment`, you will be calling an instance of `CommentQuery` instead of `ActiveQuery`.
 
+You can customize the `CommentQuery` class in many creative ways to improve your query building experience. For example,
+you can define new query building methods:
+````php
```

Comment:
Need 3 ticks, not 4 for code blocks. Also there should be empty line before code.